<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SideWidgetFileSystem</class>
 <widget class="QWidget" name="SideWidgetFileSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Object Scope &amp; Context Details</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QLineEdit" name="txtCurrentPath">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="btnBrowse">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTreeView" name="treeView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
